src/Interpreters/ClusterProxy/SelectStreamFactory.cpp

```diff
@@ -63,18 +65,15 @@ void SelectStreamFactory::createForShard(
 
     auto emplace_local_stream = [&]()
     {
-        local_plans.emplace_back(createLocalPlan(query_ast, header, context, processed_stage, shard_info.shard_num, shard_count, /*coordinator=*/nullptr));
+        local_plans.emplace_back(createLocalPlan(query_ast, header, context, processed_stage, shard_info.shard_num, shard_count, 0, 0, /*coordinator=*/nullptr));
```

Comment:
Add comments for hardcoded values like for `coordinator`. One of the builds in CI catch unmatched argument names so I think it's good rule to enforce.